Resources/Prototypes/_Impstation/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -0,0 +1,14 @@
+- type: entity
+  parent: [ClothingOuterStorageBase, AllowSuitStorageClothing, BaseCommandContraband]
+  id: ClothingOuterCoatCMO
+  name: chief medical officer's prestige labcoat
+  description: A Medical Labcoat for the Chief Medical Officer. The high collar acts as a splash guard for when a patient's veins are particularly leaky. 
+  components:
+  - type: Sprite
+    sprite: _Impstation/Clothing/OuterClothing/Coats/cmoprestigecoat.rsi
+  - type: Clothing
+    sprite: _Impstation/Clothing/OuterClothing/Coats/cmoprestigecoat.rsi
+ # - type: Armor
```

Comment:
if they're gonna be commented, they can be deleted to save a little space

Reply:
I leave it with the hope in my heart someone figures out why it fucking breaks and then we can put it back in and joy and love will return. Inherit my legacy code boy (Should I just put a 'Todo' in this I dunno the comment ettiquite